src/ffi/defaults_builder.rs

```diff
@@ -0,0 +1,114 @@
+use std::sync::Arc;
+
+use parking_lot::Mutex;
+use telio_model::features::{FeatureLana, FeatureValidateKeys, Features};
+
+pub struct FeaturesDefaultsBuilder {
+    config: Mutex<Features>,
+}
+
+impl FeaturesDefaultsBuilder {
+    pub fn new() -> Self {
+        let config = Features {
+            wireguard: default(),
+            validate_keys: default(),
+            firewall: default(),
```

Comment:
I wonder about `firewall`, as there is no `.enableFirewall...()`. There is a `bool` which should get enabled by default on android and disabled on apple: https://github.com/NordSecurity/libtelio/blob/main/src/libtelio.udl#L601.

Could we add `.enableFirewallConnectionReset(..)` or similar? :pray: :pray: :pray:

Reply:
Yeh was not sure about subfeatures

Reply:
So will add